### ISSUE TITLE
Technical feedback: Add raw license on GitHub + link to GitHub from the book

### ISSUE THREAD

**Issue by AbcSxyZ** [pearcej/opensource#312]
It could be great to add the raw license on GitHub to know the used license from the repository in a standard manner. Next to it in the textbook, the section «  Open Source Licensing  » may be confusing, as you may expect to find things around open source licensing. A title like license alone or book license may be more appropriate ?

I'm not sure that the link between the book and the GitHub repository has been made. If it exists, it's not obvious. Could be great to have a better relation between them to have an easier navigation between the book and the repo.

**Comment by pearcej** [comment]
I agree that this is needed.

**Comment by AbcSxyZ** [comment]
It may be interesting to add it through some contributing guidelines (ex: contribute section in front matter ?)

**Comment by pearcej** [comment]
I would love this to become an open source communtiy book that serves many needs!

**Comment by schalkneethling** [comment]
@AbcSxyZ and @pearcej - I assume you are also referring to the `LICENSE` file at the project root here on GitHub? At the moment there is no license to indicate the licensing terms of the content of the repository.

**Comment by schalkneethling** [comment]
@pearcej Looking here https://github.com/pearcej/opensource/blob/main/source/ch1_oss.ptx it seems that the license should be CCBY4.0 - If this is correct, I can open up a PR to add it. While I am doing that, how about adding a `CODE_OF_CONDUCT`? Perhaps contributor covenant? https://www.contributor-covenant.org/

**Comment by AbcSxyZ** [comment]
> I assume you are also referring to the LICENSE

Yes, and there is clearer indication of licensing in the « [Open Source Licensing](https://pearcej.github.io/opensource/meta_frontmatter-2.html) » which state it for the whole project  [**CC BY-SA**, not CC BY]. Should this section be renamed too to avoid confusion, like by being called "license" alone ?  Even more with a chapter 5 being also called Open Source Licensing. (+ "open license" rather than "open source license" ?)

> there is no license to indicate the licensing terms of the content of the repository

I was wondering how a license on an end resource apply to its sources, and honestly I don't know ! As there is some distinct files in the repo, not sure what is covered by the license and what may be considered as unlicensed. It would also clarify the situation.
